client/components/GameListEntry.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 const GameListEntry = props => (
   <div 
-    className="game-entry" 
-    onMouseEnter={props.handleGameMouseEntry}
+    className={props.game.title === props.selectedGame.title ? 'current-game' : 'game-entry'}
+    onMouseEnter={(game => props.handleGameMouseEntry(game)).bind(null, props.game)}
```

Comment:
It's kinda hard to parse what's going on here, I had to do a lot of back and forth through files to sort out what was happening.

Rather than using another `.bind()` around an anonymous function enclosed in parens, I'd recommend just creating a named wrapper arrow function above your `const GameListEntry` assignment which simply accepts a `game` argument and a `callback`. Then just have it execute the callback (which you'd be passing in as `props.handleGameMouseEntry`) with `game` as the sole argument.

If this pattern is used elsewhere you could just stick it in a module and import it where needed across Components.

Reply:
implemented change